girder/api/describe.py

```diff
@@ -478,11 +477,28 @@ def listResources(self, params):
         # Definitions Object
         definitions = dict(**docs.models[None])
 
+        def _walkTree(node, path=()):
```

Comment:
Inserting this here makes the cyclometric complexity check fail.  Maybe hoist it out of the methd?

Reply:
Done :)